formidling/src/main/resources/META-INF/persistence.xml

```diff
@@ -0,0 +1,48 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
Trenger vi en egen persistence.xml for modulen? Må ikke den da ha et annet navn enn den som allerde finnes ("pu-default")

Reply:
fjernet!